<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortFilterSettingsWidget</class>
 <widget class="QWidget" name="PortFilterSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QGroupBox" name="m_pGrpMidiChannelMessageFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Channel Messages to Filter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableView" name="m_pTblMidiChannelMessageFilter"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="m_pGrpMidiControllerFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Configure MIDI Controller Messages to Filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0">
      <item row="0" column="0">
       <widget class="QTableView" name="m_pTblMidiControllerFilter"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grprMidiSystemMessagesFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>System Messages to Filter </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiResetEvents">
         <property name="accessibleName">
          <string comment="reset"/>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiActiveSensingEvents">
         <property name="accessibleName">
          <string extracomment="active sensing"/>
         </property>
         <property name="text">
          <string>Active Sensing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiRealtimeEvents">
         <property name="accessibleName">
          <string comment="realtime events"/>
         </property>
         <property name="text">
          <string>Realtime Events</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiTuneRequestEvents">
         <property name="accessibleName">
          <string comment="tune request"/>
         </property>
         <property name="text">
          <string>Tune Request</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiSongSelectEvents">
         <property name="accessibleName">
          <string comment="song select"/>
         </property>
         <property name="text">
          <string>Song Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiSongPositionPointer">
         <property name="accessibleName">
          <string comment="song position pointer"/>
         </property>
         <property name="text">
          <string>Song Position Pointer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiTimeCodeEvents">
         <property name="accessibleName">
          <string comment="midi time code"/>
         </property>
         <property name="text">
          <string>Midi Time Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pCbFilterMidiSysexEvents">
         <property name="accessibleName">
          <string comment="system exclusive"/>
         </property>
         <property name="text">
          <string>System Exclusive</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
